book/A-git-in-other-environments/sections/sublimetext.asc

```diff
@@ -0,0 +1,12 @@
+=== Git in Sublime Text
+
+From version 3.2 onwards, Sublime Text has git integration in the editor.
+
+The features are:
+*  The sidebar will show the git status of files and folders with a badge/icon.
+*  Files and folders that are in your .gitignore file will be faded out in the sidebar.
+*  In the status bar, you can see the current git branch and how many modifications you have made.
+*  All changes to a file are now visible via markers in the gutter.
+*  You can use part of the Sublime Merge git client functionality from within Sublime Text. (This requires that Sublime Merge is installed.)
```

Comment:
Two things:

1. Let's move the parenthetical to a new line to keep with our source code conventions.
1. Can you link "Sublime Merge" to [its marketing site](https://www.sublimemerge.com/)?

Reply:
Done.

Reply:
Was this newline how you wanted it? I'm unsure, asciidoctor's documentation wasn't clear on a newline after a bullet point...

Reply:
This is a bit odd. We use this markup style in [this other section](https://github.com/progit/progit2/blob/15c51bb92dba00b14d343ac52a913ff89f07ff6a/book/07-git-tools/sections/credentials.asc#L17) and it renders fine, but here the GitHub renderer is choking on the formatting. Not sure what the difference is.